mozilla-release/mobile/android/base/java/org/mozilla/gecko/tabs/TabsLayoutAdapter.java

```diff
@@ -32,6 +33,8 @@
     // Click listener for the close button on itemViews.
     private final Button.OnClickListener closeOnClickListener;
 
+    private boolean isLightTheme;
```

Comment:
`mIsLightTheme`. Are we planning to add a boolean for every other theme we implement or are we using numeric coding?